gui/wxpython/gui_core/pyedit.py

```diff
@@ -333,20 +333,18 @@ def _writeFile(self, file_path, content, additional_err_message=""):
             GError(
                 message=_(
                     "Permission denied <{}>. Please change file "
-                    "permission for writing.{}".format(
-                        file_path,
-                        additional_err_message,
-                    ),
+                    "permission for writing.{}"
+                ).format(
+                    file_path,
+                    additional_err_message,
                 ),
```

Comment:
Not having numbered or named placeholders makes it impossible in translations to reorder the placeholders to accommodate the different languages. My initial suggestion in this PR was adressing that

Reply:
I agree. Maybe this is for a different PR. There we can more precisely evaluate the impact of that particular change to existing translations.